app/views/emails/invoice_messenger_unsuccessfully_charged_for_buyer.text.liquid

```diff
@@ -1,11 +1,17 @@
-Dear {{account.name}},
+Dear {{ account.name }},
 
 Thank you for using our service.
 
-We'd like to inform you that unfortunately your last payment failed to process correctly. We will try again in 3 to 4 days. In the meantime, please update the credit card details on the Account tab of your Dashboard ({{payment_url}}) if you believe the card details are out of date.
+We're sorry to inform you that your last payment was declined.
+This may have been caused by a few common reasons:
 
-If you need any help, please don't hesitate to contact us at {{ provider.finance_support_email }}.
+- A new authentication policy enforced by your bank
```

Comment:
Not mentioning SCA explicitly ?

Reply:
I agree, we should mention it!

Reply:
This was already discussed during the design phase. I don't mind to open this discussion again but it will slow us down.

Reply:
Confirmed that we will keep this text non technical and generic.